flink-connector-vitess-cdc/src/main/java/com/ververica/cdc/connectors/vitess/VitessSource.java

```diff
@@ -86,9 +91,71 @@ public Builder<T> keyspace(String keyspace) {
             return this;
         }
 
-        /** VTCtld server config. */
-        public Builder<T> vtctldConfig(VtctldConfig vtctldConfig) {
-            this.vtctldConfig = vtctldConfig;
+        /**
```

Comment:
Looks like we pick some features

Reply:
These changes belong to [vitess] Migrate connector properties to debezium 1.9.7 (#2260). It's ok to merge this commit.

Reply:
I understand your concern and drop this commit.